nestkernel/nest_types.h

```diff
@@ -32,6 +32,19 @@
 // Generated includes:
 #include "config.h"
 
+#ifdef _OPENMP
+  #define NEST_PARALLEL_FOR_THREAD(t) \
+    _Pragma("omp parallel") \
+    { \
+      size_t t = kernel().vp_manager.get_thread_id();
```

Comment:
Yes, 100% agree, was just a matter of habit. It's quite annoying to keep in mind `thread` `index` etc.